### ISSUE TITLE
Q: should we plan for single EPUB3.epub?

### ISSUE THREAD

**Issue by GoogleCodeExporter** [w3c/epub-specs#31]
```
We have more leaf docs in EPUB3 - at a minimum we get Media Overlays, plus 
non-normative Overview and Changes docs. NCX may be a separate doc and the 
current draft organization has a separate "EPUB Publications 3.0" spec. This is 
still a bit fluid but one thing that might help resolve the inherent tension 
between separate leaf specs being good from a modularity and potential reuse 
perspective, but not necessarily easy to navigate, would be to decide not to 
create non-normative EPUBs for each individual spec, but to have a single 
non-normative EPUB3.epub document. Each individual spec could be a logical 
chapter (and concrete spine item). This kind of makes sense to me as while our 
current specs are each "books" in DocBook with multiple "chapters", these are 
really more like sections.

Basically if most people get EPUB3.epub as their single docset, with an 
umbrella TOC in NCX form and the Overview coming first as roadmap, it seems 
like it wouldn't matter as much how we sliced the salami of the other specs.

We could also in that case have acknowledgments and contributors live in one 
place not every leaf document. This information takes up quite a bit of space.

EPUB generation is low priority for now but deciding now what we want to do 
later may help us take final spec granularity decisions sooner.
```

Original issue reported on code.google.com by `whmc...@gmail.com` on 28 Oct 2010 at 5:14
